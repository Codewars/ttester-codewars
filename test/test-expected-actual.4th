\ Copyright 2019 nomennescio
s" Expected results and actual results tests" describe#{
  s" Match" it#{
     <{ 0 1 2 3 -> 0 1 2 3 }>
  }#
  s" Mismatch" it#{
     <{ 0 1 2 3 -> 4 5 6 7 }>
  }#
}#