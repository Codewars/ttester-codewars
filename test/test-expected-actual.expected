
<DESCRIBE::>Expected results and actual results tests

<IT::>Match

<PASSED::>Test Passed

<COMPLETEDIN::>0.039 ms

<IT::>Mismatch

<FAILED::>Expected 4 5 6 7 , got 0 1 2 3 

<COMPLETEDIN::>0.067 ms

<COMPLETEDIN::>0.296 ms
